src/NuGet/EntityFramework.CodeTemplates.CSharp/EntityFramework.CodeTemplates.CSharp.nuspec

```diff
@@ -4,12 +4,12 @@
     <id>EntityFramework.CodeTemplates.CSharp</id>
     <version>$NuGetPackageVersion$</version>
     <title>EntityFramework.CodeTemplates.CSharp</title>
-    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=386613</iconUrl>
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
     <tags>Microsoft EF EntityFramework</tags>
     <projectUrl>http://go.microsoft.com/fwlink/?LinkID=$ProjectUrlFwLinkID$</projectUrl>
     <licenseUrl>http://go.microsoft.com/fwlink/?LinkID=$LicenseUrlFwLinkID$</licenseUrl>
```

Comment:
Get rid of this by switching to use `$CommonMetadataElements$` for the majority of the properties. That is, remove everything set in https://github.com/dotnet/arcade/blob/dd885745172424fc8df97295514f693efaec8d24/src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.targets#L102-L118 and add `$CommonMetadataElements$`. Compare the final result 😺

Reply:
How does this package get built? This one & EntityFramework.CodeTemplates.VisualBasic don't get built by `build -pack`, and I only see those two namespaces being referenced by their own `.nuspec` files.

Reply:
These don’t ship